Makefile

```diff
@@ -179,27 +185,36 @@ package: sdist check-sdist-local manylinux-wheels check-wheel-local
 # Build the AWS lambda layer.
 # temporary target directory for AWS Lambda build artifacts
 # TODO cp39 and cp310
-target_dir := "./tmp/python"
-aws-lambda: wrapper
+target_dir := "./tmp"
+aws-lambda: check-zip wrapper
 	@if [ -f ./dist/solarwinds_apm_lambda.zip ]; then \
 		echo -e "Deleting old solarwinds_apm_lambda.zip"; \
 		rm ./dist/solarwinds_apm_lambda.zip; \
 	 fi
 	rm -rf ./tmp
 	@echo -e "Creating target directory ${target_dir} for AWS Lambda layer artifacts."
-	mkdir -p ${target_dir}
+	mkdir -p ${target_dir}/python
+	@echo -e "Install upstream dependencies to include in layer"
+	@/opt/python/cp38-cp38/bin/pip3.8 install -t ${target_dir}/python -r lambda/requirements.txt
+	@echo -e "Install upstream dependencies without deps to include in layer"
+	@/opt/python/cp38-cp38/bin/pip3.8 install -t ${target_dir}/nodeps -r lambda/requirements-nodeps.txt --no-deps
 	@echo -e "Install solarwinds_apm to be packed up in zip archive to target directory."
-	@/opt/python/cp38-cp38/bin/pip3.8 install . -t ${target_dir}
+	@/opt/python/cp38-cp38/bin/pip3.8 install . -t ${target_dir}/nodeps --no-deps
```

Comment:
We install `solarwinds_apm` and some deps with `--no-deps` and in a separate folder else not all packages will be fully downloaded and accessible at `opentelemetry/instrumentation` path ([this comment](https://github.com/solarwinds/opentelemetry-lambda/blob/main/python/src/otel/Dockerfile#L12-L13)).